test/check-bin.sh

```diff
@@ -70,6 +70,18 @@ expectOut $? 0 "tslint with with -r pointing to custom rules did not find lint f
 ./bin/tslint -c test/config/tslint-almost-empty.json src/tslint.ts
 expectOut $? 0 "-c relative path without ./ did not work"
 
+# make sure calling tslint with a config file which extends a package relative to the config file works
+./bin/tslint -c test/config/tslint-extends-package-no-mod.json src/tslint.ts
+expectOut $? 0 "tslint (with config file extending relative package) did not work"
+
+# check that a tslint file (outside of the resolution path of the tslint package) can resolve a package that is
+# installed as a dependency of tslint. See palantir/tslint#1172 for details.
+tmpDir=$(mktemp -d)
```

Comment:
On the version of `mktemp` installed on my Mac, this is unfortunately an invalid usage. I'm okay with getting rid of this test here though, as you have a similar test below. Plus your test below should work on Windows also, which is great.